proposals/nnnn-actors.md

```diff
@@ -589,12 +589,14 @@ By default, the mutable stored properties (declared with `var`) of an actor clas
 All actor classes conform to a new protocol `Actor`:
 
 ```swift
-protocol Actor: AnyObject {
-  func enqueue(partialTask: PartialAsyncTask)
-}
+protocol Actor: AnyObject { /* unspecified */ }
 ```
 
-The `enqueue(partialTask:)` operation is a low-level operation used to queue work for the actor to execute. `PartialAsyncTask` represents a unit of work to execute. It effectively has a single synchronous function, `run()`, which should be called synchronously within the actor's context. Only the compiler can produce new `PartialAsyncTasks`. To explicitly enqueue work on an actor, use the `run` method:
+All actor classes implicitly conform to the `Actor` protocol. Non-actor classes cannot conform to the `Actor` protocol.
+
+The `Actor` protocol has no stated requirements in this proposal. Rather, a separate proposal on customized executors will specify the requirements and detail how to provide an alternative implementation of an actor's queueing behavior. Regardless of those details, an actor that does not provide implementations of those requirements will receive synthesized default implementations. Such actors are said to be default actors.
+
+The `Actor` protocol provides a single operation, the `run` method, which executes a given function on this actor:
```

Comment:
Oh you actually mean to provide it built in?

I thought this was an example "if you want to, you can put such run() onto your actor".

- Do all actors really need to be thrown a random closure at them?
- The boilerplate of adding such function is not that high if we don't pre-define it, anyone who wants their actor to run random stuff can simply define such function on it.
  - they are also then in charge of it being `public` or `internal` which may matter; I would not want to allow anyone to run a closure which may block and do terrible things on my "framework internal actor" even if the actor is public... so at least not pre-defining this function would be better imo.
  -